os_installer2/diskman.py

```diff
@@ -199,7 +199,7 @@ def build_ext_space(self):
             if self.min_size < self.usedspace:
                 self.min_size = self.usedspace
             self.resizable = True
-            break
+            break        
```

Comment:
...?

Reply:
Sorry, not sure how that one slipped by.